src/gui/widgets/InstrumentFunctionViews.cpp

```diff
@@ -137,6 +139,22 @@ InstrumentFunctionArpeggioView::InstrumentFunctionArpeggioView( InstrumentFuncti
 			"number of octaves." ) );
 
 
+	m_arpSkipKnob->setLabel( tr( "SKIP" ) );
+	m_arpSkipKnob->setHintText( tr( "Skip rate:" ) + " ", " " + tr( "%" ) );
```

Comment:
Fixed! I have not clue as to what precisely I was thinking when laying out the text. The Gate knob already has the layout I needed?